### PR TITLE
feature-29(network interfaces)

### PR DESCRIPTION
**Pull request by Koufan-De-King**
Developer: Koufan King

Feature-29 was done by adding a new service in the 'services' folder named 'network_service.py' through which information about the systems' network interfaces is retrieved and then reported into the app.py file which invokes the 'jsonify' utility to displayed the obtained information into a graph on the browser.

### CODE REVIEW

File: __pycache__/app.cpython-310.pyc
Comment (stephane-segning) [inline __pycache__/app.cpython-310.pyc:1]: You don't need this file. Please remove it

File: services/__pycache__/cpu_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/cpu_service.cpython-310.pyc:1]: You don't need this file. Please remove it

File: services/__pycache__/memory_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/memory_service.cpython-310.pyc:1]: You don't need this file. Please remove it

File: services/__pycache__/network_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/network_service.cpython-310.pyc:1]: You don't need this file. Please remove it

File: services/__pycache__/time_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/time_service.cpython-310.pyc:1]: You don't need this file. Please remove it

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Perfect Job 😎!
Just need to remove not-useful files and your done

File: __pycache__/app.cpython-310.pyc
Comment (Koufan-De-King) [reply __pycache__/app.cpython-310.pyc:1]: > You don't need this file. Please remove it

Okay sir, I'll do that now

**Comment by Koufan-De-King** [comment]
> Perfect Job 😎! Just need to remove not-useful files and your done

**Comment by Koufan-De-King** [comment]
> > Perfect Job 😎! Just need to remove not-useful files and your done

Thanks sir. 
I have deleted the unwanted files. I don't know if you can see that.

**Review by stephane-segning — APPROVED** [review]

**Comment by Koufan-De-King** [comment]
Merge complete!

**Pull request merged**